lib/nextstrain/sphinx/theme/footer.html

```diff
@@ -32,7 +32,7 @@
     {% macro TeamMember(name, image, link) %}
     <span>
       {% if link %}<a href="{{ link |e }}">{% endif %}
-        <img src="{{ pathto('_static/team/' + image, 1) }}" width="32"/>
+        <img src="https://raw.githubusercontent.com/nextstrain/nextstrain.org/HEAD/static-site/static/team/{{ image }}" width="32"/>
```

Comment:
> Ideally they could be referenced by nextstrain.org/static/*.jpg, but the filenames are not easily determinable like the GitHub URLs.

We could presumably fix this with some Gatsby config or something? (Or additional app routes.)